src/modules/tokens/views/Accounts/AccountDetails.tsx

```diff
@@ -234,6 +232,7 @@ export const AccountDetails: () => JSX.Element = () => {
       key: balance.pool + ':' + balance.tokenIndex,
       columns: [
         { value: balance.tokenIndex },
+        { value: balance.uri },
```

Comment:
Should we use the `HashPopover` component for this? Might be useful if it's something they will copy a bunch.